chemclipse/plugins/org.eclipse.chemclipse.msd.identifier.supplier.nist/src/org/eclipse/chemclipse/msd/identifier/supplier/nist/core/support/Identifier.java

```diff
@@ -140,9 +136,6 @@ public IMassSpectra runMassSpectrumIdentification(List<IScanMSD> massSpectrumLis
 					long maxProcessTime = (long)(searchSettings.getTimeoutInMinutes() * IChromatogramOverview.MINUTE_CORRELATION_FACTOR);
 					logger.info("Max Process Time: " + maxProcessTime);
 					logger.info("Run Identification");
-					Compounds compounds = runNistApplication(runtimeSupport, maxProcessTime, waitTime, monitor);
-					logger.info("Assign Compounds");
```

Comment:
Is it intended to skip the identification process?

Reply:
You are right. I removed too much. Only the `identificationResults` reassigned is unused.